torchvision/models/inception.py

```diff
@@ -114,7 +114,8 @@ def forward(self, x):
         # 8 x 8 x 2048
         x = self.Mixed_7c(x)
         # 8 x 8 x 2048
-        x = F.avg_pool2d(x, kernel_size=8)
+        adaptiveAvgPoolWidth = x.shape[2]
+        x = F.avg_pool2d(x, kernel_size=adaptiveAvgPoolWidth)
```

Comment:
isn't it simpler to just use `F.adaptive_avg_pool2d(x, 1)` here?

Reply:
You are right that would be better. Should I make this change in my PR?

Reply:
Please have a look at the new PR (https://github.com/pytorch/vision/pull/744#issue-252737658)